lib/dom.js

```diff
@@ -3568,6 +3568,38 @@ declare class ClientRectList { // extension
   [index: number]: ClientRect;
 }
 
+declare class DOMRectReadOnly {
```

Comment:
Seems like these classes are already declared in [`bom.js`](https://github.com/facebook/flow/blob/master/lib/bom.js#L395)?

Reply:
Oh I didn't realize it.
I see, surely that's not DOM things BOM is property place.

next my plan is 
1. fix Exists DOMRect definition based on @mroch review.
2. add DOMRect(already declared in bom.js) to getBoundingClient() return type.

Reply:
@fishythefish I moved these classes to bom.js!